static/templates/composer.tpl

```diff
@@ -1,4 +1,4 @@
-<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable -->">
+<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable --><!-- IF isTopicOrMain --> topic-or-main<!-- ENDIF isTopicOrMain -->">
```

Comment:
It's a class so you can hide things based on if the composer is composing a
main post.

On Aug 4, 2016 07:31, "Julian Lam" <notifications@github.com> wrote:

> In static/templates/composer.tpl
> <https://github.com/NodeBB/nodebb-plugin-composer-default/pull/59#discussion_r73521691>
> :
>
> > @@ -1,4 +1,4 @@
> > -<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable -->">
> > +<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable --><!-- IF isTopicOrMain --> topic-or-main<!-- ENDIF isTopicOrMain -->">
>
> What's this you're sneaking in here? 😛
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NodeBB/nodebb-plugin-composer-default/pull/59/files/83d4aee42c2ce1327239a9572aa3ea992abb17eb#r73521691>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAxDddudRsZkFQi0SUKnvPXhFj166KwSks5qcem5gaJpZM4JcJ-c>
> .
>